docs/testing/01-overview.md

```diff
@@ -88,6 +88,26 @@ export const config: Config = {
 };
 ```
 
+### Command Line Arguments
+
+While the Stencil CLI offers a certain set of command line flags to specify e.g. which types of tests to run, you also have access to all Jest options through the CLI. For example to specify a single test, you can pass in a positional argument to the command, e.g.:
+
+```sh
+# run a single unit test
+npx stencil test --spec src/components/my-component/my-component.spec.ts
```

Comment:
I think you still need the argument separator (`--`) here. On a sample project (`npm init stencil@latest component link-test`) that has one additional component (`npm run generate`, enter 'link-me' for the test name), this runs all of the tests. If I run this command with `--` to denote the file path as a positional arg, it runs just those tests:
```
➜  link-test git:(main) ✗ npx stencil test --spec -- src/components/my-component/my-component.spec.ts
[34:34.3]  @stencil/core
[34:34.4]  v4.9.1 🍬
[34:34.5]  testing spec files
[34:34.5]  jest args: --spec --max-workers=8 -- src/components/my-component/my-component.spec.ts
 PASS  src/components/my-component/my-component.spec.ts
  my-component
    ✓ renders (8 ms)
    ✓ renders with values (2 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        0.241 s, estimated 1 s
Ran all test suites matching /src\/components\/my-component\/my-component.spec.ts/i.

➜  link-test git:(main) ✗ npx stencil test --spec src/components/my-component/my-component.spec.ts
[34:41.9]  @stencil/core
[34:42.1]  v4.9.1 🍬
[34:42.1]  testing spec files
[34:42.1]  jest args: --spec --max-workers=8
 PASS  src/components/my-component/my-component.spec.ts
 PASS  src/components/link-me/test/link-me.spec.tsx
 PASS  src/utils/utils.spec.ts

Test Suites: 3 passed, 3 of 5 total
Tests:       7 passed, 7 total
Snapshots:   0 total
Time:        0.341 s, estimated 1 s
Ran all test suites.
```

Reply:
> I think you still need the argument separator (`--`) here.

Sorry I was meaning to remove `--spec` here to showcase positional arguments

Reply:
@rwaskiewicz thanks, I forgot about that when restructuring the docs. Pushed an update.